layouts/partials/collapsible-menu.html

```diff
@@ -21,38 +21,62 @@
-    <div class="collapsible-menu container hidden-lg hidden-md visible-sm visible-xs">
-      <div class="container collapsible-menu-item related-content">
+    <div class="{{ $classname }} margin-top-10 container hidden-lg hidden-md visible-sm visible-xs">
+      <div class="container padding-left-0 padding-right-0 {{ $classname }}-item related-content">
         <a href="#sidebars-contents" data-toggle="collapse" aria-expanded="false" aria-controls="sidebars-contents">
           <div class="col-xs-6"><i class="fa fa-plus fa-bars"></i></div>
-          <div class="col-xs-18 collapsible-menu-text">Related Content</div>
+          <div class="col-xs-18 {{ $classname }}-text">Related Content</div>
         </a>
-      </div>  
-      <div class="container collapse" id="sidebars-contents">
+      </div>
+      <div class="container collapse {{ if eq $classname "sidebar-block" }} padding-left-0 padding-right-0 {{ end }}" id="sidebars-contents">
       {{ range .Children }}
         {{ if eq .HasChildren false }}
-          <div class="row collapsible-menu-item{{ if eq $currentPage.Title .Name }} active{{ end }}">
+          <div class="row {{ $classname }}-item{{ if eq (lower $currentPage.Title) (lower .Name) }} active{{ end }}">
             <a href="{{ .URL }}" {{ if eq $currentPage.Title .Name }}class="active"{{ end }}>
-              <div class="col-xs-18 col-xs-offset-6 collapsible-menu-text">{{ .Name }}</div>
+              <div class="col-xs-6">{{ .Pre }}</div>
+              <div class="col-xs-18 {{ $classname }}-text">{{ .Name }}</div>
             </a>
           </div>
         {{ end }} <!-- if eq .HasChildren false end -->
         {{ if .HasChildren }}
-          <div class="row collapsible-menu-item{{ if eq $currentPage.Title .Name }} active{{ end }}">
+          <div class="row {{ $classname }}-item{{ if eq (lower $currentPage.Title) (lower .Name) }} active{{ end }}">
             <a href="#sub-children-{{ .Identifier }}" data-toggle="collapse" aria-expanded="false" aria-controls="sub-children{{ .Identifier }}" {{ if eq $currentPage.Title .Name }}class="active"{{ end }}>
               <div class="col-xs-6">{{ .Pre }}</div>
-              <div class="col-xs-18 collapsible-menu-text">{{ .Name }}</div>
+              <div class="col-xs-18 {{ $classname }}-text">{{ .Name }}</div>
             </a>
           </div>
-          <div class="row collapsible-menu-sub-items collapse" id="sub-children-{{ .Identifier }}">
+          <div class="row {{ $classname }}-sub-items collapse" id="sub-children-{{ .Identifier }}">
+            {{ if .URL }}
+              <div class="row {{ $classname }}-sub-item{{ if eq (lower $currentPage.Title) (lower .Name) }} active{{ end }}">
+                <a href="{{ .URL }}" {{ if eq (lower $currentPage.Title) (lower .Name) }}class="active"{{ end }}>
+                  <div class="col-xs-18 col-xs-offset-6 {{ $classname }}-text">{{ .Name }} Home</div>
```

Comment:
Why are we innjecting the term 'home' into the label of a sidebar item? We should display it as intended in the menu definition

Reply:
I was expecting to go to the parent page of the collapsible parent, like the following screenshot. 

Do you think it's okay to have the repeated names without a `home` term? 

![image](https://user-images.githubusercontent.com/39588094/109699932-2bb00e80-7b5f-11eb-8775-f78f6cbedff0.png)

Reply:
Currently, if you want that you add it to the sub menu manually rather than have it added artificially. I think we should remove that line, as injecting additional entries can be detrimental

Reply:
But if adding it manually, it generates duplicates on desktop view. while on desktop view, the expected behavior is the parent is clickable, please refer to https://iot.eclipse.org/community/, the resource sidebar.

Reply:
I think it makes sense here to inject a link in the mobile link only since the top level link won't link to the page.

However, please drop the "Home" suffix. I don't think it's needed.

Reply:
Dropped "Home" text.